package.json

```
@@ -134,37 +128,30 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render dist:js build:api test build:markdownDocs bundler postinstall",
-    "build:ci": "npm-run-all sweep build",
+    "build": "npm-run-all build:sass sass:render cssLint dist:js bundler postinstall build:api types",
     "build:api": "wca analyze 'src/auro-dropdown.js' --outFiles docs/api.md",
-    "build:demo": "npm-run-all build demo:rm:build demo:new:build demo:copy:index demo:copy:demo demo:update:index",
-    "build:dev:assets": "npm-run-all build:sass:demo build:sass:component postCss:component sass:render build:docs",
-    "build:markdownDocs": "node scripts/generateDocs.js",
-    "build:docs": "npm run build:markdownDocs",
-    "build:sass": "npm-run-all build:sass:demo build:sass:component postCss:component sass:render",
-    "build:sass:demo": "sass --no-source-map demo:demo",
+    "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render",
+    "build:sass": "npm-run-all build:sass:component postCss:component sass:render",
+    "build:sass:dev": "npm-run-all build:sass:component postCss:component sass:render",
+    "build:sass:demo": "sass ./demo/sass/ --output ./demo/css/",
     "build:sass:component": "sass --no-source-map src:src",
-    "build:watch": "nodemon -e scss,js,html --watch src --watch apiExamples --exec npm run build:dev:assets",
+    "build:watch": "nodemon -e scss,js --watch src --exec npm run build:dev:assets",
```

```suggestion
    "build:watch": "nodemon -e scss,js,html --watch src --watch apiExamples --exec npm run build:dev:assets",
```

Undo this change. The previous script was correct.